app/controllers/lessons_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class LessonsController < ApplicationController
```

Comment:
We need to make actions new, create, edit, update only for the authorized users.